narwhals/_dask/namespace.py

```diff
@@ -85,6 +87,23 @@ def sum(self, *column_names: str) -> DaskExpr:
             backend_version=self._backend_version,
         ).sum()
 
+    def len(self) -> DaskExpr:
+        dd = get_dask_dataframe()
+        pd = get_pandas()
+
+        # coverage bug? this is definitely hit
+        return DaskExpr(  # pragma: no cover
+            lambda df: [
+                dd.from_pandas(pd.Series([len(df._native_dataframe)], name="len"))["len"]
+            ],
```

Comment:
I am already late to the party, but maybe we can keep this lazy with:

```py
lambda df: [
    df._native_dataframe.loc[:, df._native_dataframe.columns[0]].size
],
```

Reply:
ooooh man, thanks! I should've waited here 😳